src/System.Text.Json/src/Configurations.props

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp-Unix;
```

Comment:
No, I will not. Should I just have `netcoreapp-Windows_NT` then? Having just `netcoreapp` fails to compile.

Reply:
It fails to compile because of you are referencing CoreLib and netcoreapp implementation directly in .csproj. This package should not be referencing CoreLib. It should be just referencing the public surface via reference assemblies. Once you fix that, you will be able to make this to just netcoreapp.

Reply:
Should I leave the uap configuration with the os specified?

Also, is it fine to keep the project reference to the `System.Runtime` ref within the `System.Text.Json` ref?

Reply:
Inside ref projects we strongly push toward having only project reference to other ref projects because the reference assemblies are the first thing we build and ordering is not guaranteed so we use project references in ref projects to make sure we build all its dependencies when building a project.

Having the uap being os specific doesn't hurt anything because uap is not supported in Unix, however we try to only use it os specific when we're going to have conditions in the csproj using $(TargetsWindows) property. Because in order for it to be set correctly the configuration needs to be os specific.